### ISSUE TITLE
new tablet file for CTE-660

### ISSUE THREAD

**Issue by leschalu** [linuxwacom/libwacom#59]
> the tablet works but does not
show up in GNOME's configuration panel

**Device name**
CTE-660

`bamboo-one-m.tablet`

```
[Device]
Name=Wacom CTE-660
DeviceMatch=usb:056a:006b
Class=Bamboo
Width=8
Height=5

[Features]
Stylus=true
```

**Comment by whot** [comment]
thank you, looks good. I think the full name is Wacom Bamboo Pen? or something? `sudo evemu-record` should tell you in the first few lines. Can you replace the `Name=` with whatever that shows and submit a PR to get this into the repo? Thanks.

Also, we need `Buttons=0`, otherwise you'll get a warning. Best to just copy the bits from the `data/bamboo-2fg-m-p.tablet` file, though afaict Reversible is false for this one because it's symmetric anyway.

**Comment by leschalu** [comment]
Pull request #60
---

**Comment by whot** [comment]
Closed by #60
